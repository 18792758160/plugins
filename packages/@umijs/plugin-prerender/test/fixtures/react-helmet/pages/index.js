import React from 'react';
import { Helmet } from 'react-helmet';

const styles = {
  normal: 'normal___1KW4T',
};

function Page(props) {
  return (
    <div className={styles.normal}>
      <Helmet>
        <title>Hello Umi SSR Helmet Home title</title>
      </Helmet>
      <h1>Page index {window.context.siteName}</h1>
      <h2>csr: {props.data && props.data.csr}</h2>
    </div>
  );
}

Page.getInitialProps = () => {
  console.log('Home getInitialProps');
  return {
    data: {
      ssr: 'http://127.0.0.1:7001',
      csr: 'http://127.0.0.1:8000',
    },
  };
};

export default Page;
