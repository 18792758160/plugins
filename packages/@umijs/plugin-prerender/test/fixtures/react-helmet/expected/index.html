<!DOCTYPE html><html lang="en" data-reactroot><head><title data-react-helmet="true">React Helmet title index</title><link rel="stylesheet" href="/"><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Default Title</title><script>window.g_useSSR=true;
  window.g_initialData = {"data":{"ssr":"http:\u002F\u002F127.0.0.1:7001","csr":"http:\u002F\u002F127.0.0.1:8000"}};</script><script>window.routerBase = "/";</script><meta data-react-helmet="true" name="description" content="React Helmet title Index description"></head><body><div id="root"><div class="normal___1KW4T"><h1>Page index <!-- -->Umi SSR react-helment</h1><h2>csr: <!-- -->http://127.0.0.1:8000</h2></div></div><script src="/umi.js"></script></body></html>
